lib/stream-utils.ts

```diff
@@ -93,3 +95,14 @@ export async function streamToJson<T>(stream: Readable): Promise<T> {
     });
   });
 }
+
+export async function streamToFile(stream: Readable): Promise<tmp.FileResult> {
+  try {
+    const dataBuffer = await streamToBuffer(stream);
+    const fileObj = tmp.fileSync();
+    writeFileSync(fileObj.fd, dataBuffer);
```

Comment:
can we use the promisified version of `writeFile` instead like in`lib/inputs/rpm/static.ts`?

Reply:
Also, do even use this function?

Reply:
Removed :)